test/js/pageobjects/DocumentUpload.js

```diff
@@ -13,12 +13,14 @@ class DocumentUpload extends Base {
   get uploaderBtn() { return this.$('.onfido-sdk-ui-Uploader-buttons')}
   getUploadInput() { return (async ()=>{
     const input = this.$('.onfido-sdk-ui-CustomFileInput-input')
-    this.driver.executeScript((el) => {
+    // eslint-disable-next-line prefer-arrow-callback
```

Comment:
Why are you disabling linter rules here?

Reply:
If I turn it into arrow function it breaks all the upload tests.